ml-package-versions.yml

```diff
@@ -188,3 +188,21 @@ fastai-1.x:
     requirements: ["scikit-learn"]
     run: |
       pytest tests/fastai/test_fastai_autolog.py --large
+
+onnx:
+  package_info:
+    pip_release: "onnx"
+    install_dev: |
+      sudo apt-get install protobuf-compiler libprotoc-dev
+      temp_dir=$(mktemp -d)
+      git clone https://github.com/onnx/onnx.git $temp_dir
+      cd $temp_dir
+      git submodule update --init --recursive
+      python setup.py install
+
+  models:
+    minimum: "1.5.0"
+    maximum: "1.8.0"
+    requirements: ["onnxmltools", "onnxruntime", "tf2onnx", "keras", "tensorflow", "scikit-learn"]
```

Comment:
Sorry I let this PR sit for so long. I don't think we need `onnxmltools`, `tf2onnx`, `keras`, or `tensorflow` as requirements anymore.

Reply:
Thanks for the catch. Removed them!